File: DCC/UMB/CDB/CDBWeb/DCC.UMB.CDB.Web/Models/WorkingUnitViewModel.cs
using DCC.UMB.WDF.Models;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Runtime.Serialization;
using System.Web;

namespace CDBModel {
    public class WorkingUnitViewModel : Working_unit {       
        [DataMember(IsRequired = true)]
        [Display(Name = "Is Root")]
        public System.Boolean Is_root {
            get {
                return this.is_root == "Yes" ? true : false;
            }
            set {
                this.is_root = value == true ? "Yes" : "No";
            }
        }

        [DataMember(IsRequired = true)]
        [Display(Name = "Is Leaf")]
        public System.Boolean Is_leaf {
            get {
                return this.is_leaf == "Yes" ? true : false;
            }
            set {
                this.is_leaf = value == true ? "Yes" : "No";
            }
        }

        [DataMember(IsRequired = true)]
        [Display(Name = "Is Staff Posted")]
        public System.Boolean Is_staff_posted {
            get {
                return this.is_staff_posted == "Yes" ? true : false;
            }
            set {
                this.is_staff_posted = value == true ? "Yes" : "No";
            }
        }
    }
}

File: DCC/UMB/CDB/CDBWeb/DCC.UMB.CDB.Web/JS/WorkingUnits/WorkingUnitViewModel.ts
/// <reference path="../app-include.ts"/>

module CDBModel {
    export class WorkingUnitViewModel extends Working_unit {

        constructor(
            public Is_root?: boolean 
            , public Is_leaf?: boolean
            , public Is_staff_posted?: boolean
            ) {
                super();                                        

                //this.Is_root = (this.is_root == "Yes" ? true : this.is_root == "No"? false :  undefined);
                //this.Is_leaf = (this.is_leaf == "Yes" ? true : this.is_leaf == "No"? false : undefined);
                //this.Is_staff_posted = (this.is_staff_posted == "Yes" ? true : this.is_staff_posted == "No" ? false : undefined);
                this.Is_root = (this.is_root == "Yes" ? true : false);
                this.Is_leaf = (this.is_leaf == "Yes" ? true : false);
                this.Is_staff_posted = (this.is_staff_posted == "Yes" ? true : false);
        }        
        static fld_Is_root: string = "Is_root";
        static fld_Is_leaf: string = "Is_leaf";
        static fld_Is_staff_posted: string = "Is_staff_posted";
    }
}